docs/get-started/alerting.md

```diff
@@ -6,64 +6,161 @@
 
 PMM includes a set of default templates with common events and expressions for alerting. You can also create your own templates if you need custom expressions on which to base your alert rules.
 
-You can check the  alert templates available for your account under **Alerting > Alert rule templates** tab. PMM lists here the following types of templates:
+You can check the alert templates available for your account under **Alerting > Alert rule templates** tab. PMM lists here the following types of templates:
 
 1. Built-in templates, available out-of-the-box with PMM.
-2. Custom templates created or uploaded on the **Alerting page > Alert Templates** Tab. 
-3. Custom template files available in your  ``yaml srv/alerting/templates`` directory. PMM loads them during startup.
-
-### Silences
-Silences specify periods of time to suppress alerts and their associated notifications. During a silence, PMM continues to track metrics and trigger alerts but does not send notifications to the specified contact points. Once the silence expires alerts and notifications will resume.
-For example, you can create a silence to suppress trivial notifications during weekends.
+2. Custom templates created or uploaded on the **Alerting page > Alert Templates** tab.
+3. Custom template files available in your ``srv/alerting/templates`` directory. PMM loads them during startup.
 
 ### Contact points
-Contact points specify how PMM should deliver Grafana-managed alerts. When an alert fires, a notification is sent to the specified contact points. 
 
-Depending on the severity of an alert, you might want to send different alerts to different channels. For example, you can deliver common alerts via Slack channel, but send an email notification for potentially critical issues. 
+Contact points specify how PMM should deliver Grafana-managed alerts. When an alert fires, PMM sends a notification to the specified contact points.
+
+Depending on the severity of an alert, you might want to send different alerts to different channels. For example, you can deliver common alerts via Slack channel, but send an email notification for potentially critical issues.  
+
+PMM uses Email as the default contact point but you can choose from a variety of contact points, including Slack, Email, Webhooks, PagerDuty, and more.
+
+Before PMM can send out email notifications via Email contact points, you will need to:
+
+1. Configure Email (SMTP) server settings to add your email account settings
+2. Configure a contact point to define the email delivery options
+
+Contact points with invalid settings show a **No Attempts** status under {{icon.bell}}  **Alerting > Contact points**.
+
+### Configure Email (SMTP) server settings
+
+To set up email notifications, edit the PMM’s **grafana.ini** file. This holds the settings for [Grafana’s Alertmanager](https://grafana.com/docs/grafana/latest/alerting/manage-notifications/alertmanager/) which routes notifications to contact points.
```

Comment:
If I remember correctly it was decided NOT to show this "interim" solution as it doesnt survive an upgrade when using Docker containers